tests/roxctl/bats-tests/cluster/central-init-bundles.bats

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bats
+
+load "../helpers.bash"
+
+output_file=""
+
+setup_file() {
+  local -r roxctl_version="$(roxctl-development version || true)"
+  echo "Testing roxctl version: '${roxctl_version}'" >&3
+
+  [[ -n "${API_ENDPOINT}" ]] || fail "Environment variable 'API_ENDPOINT' required"
+  [[ -n "${ROX_ADMIN_PASSWORD}" ]] || fail "Environment variable 'ROX_ADMIN_PASSWORD' required"
+}
+
+setup() {
+  output_file="$(mktemp -d -u)"
+}
+
+teardown() {
+  rm -rf "${output_file}"
+}
+
+@test "roxctl central init-bundles fetch-ca" {
+  run roxctl_authenticated central init-bundles fetch-ca --output ${output_file}
+  assert_success
+  assert_file_exist "${output_file}"
+}
```

Comment:
Tomek, this is still prone to failures...
![9phngw](https://github.com/user-attachments/assets/e4de404d-941f-4464-9a84-658eb5248dc9)

Imagine the following situation:

```
$ ls /tmp
.
..
dir1
dir2


$ mktemp -d -u
/tmp/dir99/blablabla


$ roxctl_authenticated central init-bundles fetch-ca --output "/tmp/dir99/blablabla"
Error: /tmp/dir99 does not exist
```

Why can't we do either:

1. Use `mktemp -d` and then append the filename?
2. Call `output_file="$(mktemp -d -u)"` followed by `mkdir -p "$output_file"?

Reply:
I think you're right and there is no need to complicate it. Let's remove `-u` from the first PR and it will be good.